src/components/SQFormDialog/SQFormDialogInner.tsx

```diff
@@ -97,18 +97,21 @@ const actionStyles = (theme: Theme) => ({
   display: 'flex',
   justifyContent: 'space-between',
   flex: '1 1 100%',
-  padding: '10px 20px',
   ...stickyStyles(theme),
   bottom: 0,
-  borderTop: `1px solid ${theme.palette.divider}`,
+  borderTop: `1px solid B3B3B3`,
```

Comment:
Hey @samasastudio, we need to have the hash '#' before the color value: `borderTop: 1px solid #B3B3B3`.  Same below on line 112.

Reply:
Oh no!  What a goof thanks @aharpt just pushed up the fix and here's an updated screenshot:
![Screenshot 2022-12-06 at 12 07 31 PM](https://user-images.githubusercontent.com/55225884/205988569-6dabc472-ef62-4fb6-bd7a-2477d538d7b3.png)

Reply:
^-- note the colors appearing for the header and footer dividers where they were not rendering before.

Reply:
@samasastudio Looks good.